### ISSUE TITLE
Make text visible

### ISSUE THREAD

**Issue by gubuntu** [kartoza/SAGTA#61]
This has been mentioned as a problem for months, why was there no issue? Things don't get fixed without an issue. 

Users struggle to read the login prompts so many are not aware that they have to login, or where to login, to access the Pro version. Please fix asap

<img width="393" alt="Screenshot 2022-07-22 at 11 26 21" src="https://user-images.githubusercontent.com/496911/180409202-afff7ed9-2b97-4921-bc93-155773e99823.png">

@NyakudyaA @seabilwe

**Comment by NyakudyaA** [comment]
Fixed by changing the default color in `/www/lizmap/www/assets/css/bootstrap.min.css` from color 777 to 
```
.navbar .nav>li{float:left}.navbar .nav>li>a{float:none;padding:10px 15px 10px;color:#3815e4
```

**Comment by gubuntu** [comment]
thanks, this is better, but ideal would be the light grey matching the subtitle text in the map page header

**Comment by gubuntu** [comment]
Blue font on blue tool bar doesn’t work.
![WhatsApp Image 2023-01-30 at 13 02 43](https://user-images.githubusercontent.com/496911/215460825-861c9a20-2d77-455d-a5b4-bbbecfaa8aea.jpeg)

**Comment by NyakudyaA** [comment]
![update](https://user-images.githubusercontent.com/2510900/215497809-a0b6d6e9-bc3d-44c9-a8c6-2d4e0e36767b.png)

Does this look better

**Comment by gubuntu** [comment]
nope, light grey (close to white) as per https://github.com/kartoza/SAGTA/issues/61#issuecomment-1372320406

**Comment by NyakudyaA** [comment]
This is fixed
![Screenshot from 2023-09-28 12-59-37](https://github.com/kartoza/SAGTA/assets/2510900/3ebf608d-1a54-4de7-89cc-52976e42789f)
